packages/blocks/src/api/raw-handling/test/utils.js

```diff
@@ -191,4 +191,40 @@ describe( 'getBlockContentSchema', () => {
 			output
 		);
 	} );
+
+	it( 'should handle properly merging of classes and attributes', () => {
```

Comment:
Some additional cases that I think could be good to test:

- Where the first block has `classes` and the second doesn't.
- Where both have classes
- Where there is a class that is in both, make sure it isn't duplicated

Reply:
Done. Found out that attributes property had duplication issue. Fixed it.